Mathlib/Algebra/Polynomial/Splits.lean

```diff
@@ -388,7 +390,7 @@ theorem mem_lift_of_splits_of_roots_mem_range [Algebra R K] {f : K[X]}
 A polynomial of degree `2` with a root splits.
 -/
 theorem splits_of_natDegree_eq_two {f : Polynomial K} {x : L} (h₁ : f.natDegree = 2)
-    (h₂ : eval₂ i x f = 0) : Splits i f := by
+    (h₂ : eval₂ i x f = 0) : Splits (f.map i) := by
```

Comment:
This should get a `eval i x f = 0 -> Splits f` version too, same for `splits_of_degree_eq_two`.